sql/connect/server/src/test/scala/org/apache/spark/sql/connect/SparkConnectServerTest.scala

```diff
@@ -70,12 +70,12 @@ trait SparkConnectServerTest extends SharedSparkSession {
     super.afterAll()
   }
 
-  override def beforeEach(): Unit = {
+  protected override def beforeEach(): Unit = {
     super.beforeEach()
     clearAllExecutions()
   }
 
-  override def afterEach(): Unit = {
+  protected override def afterEach(): Unit = {
```

Comment:
keep consistency with signature of `BeforeAndAfterEach ` and fix an inheritance error when introducing `StorageRootMixin`